lib/entities/quote/DutchQuote.ts

```diff
@@ -358,15 +358,19 @@ export abstract class DutchQuote<T extends DutchQuoteRequest> implements IQuote
   }
 
   // Returns the number of gas units extra required to execute this quote through UniswapX
-  static getGasAdjustment(classicQuote: ClassicQuote): BigNumber {
+  static getGasAdjustment(classicQuote: ClassicQuote, adjustmentBps?: number): BigNumber {
     let result = BigNumber.from(0);
 
     // fill contract must unwrap WETH output tokens
     if (classicQuote.request.info.tokenOut === NATIVE_ADDRESS) {
       result = result.add(WETH_UNWRAP_GAS);
     }
 
-    return result.add(UNISWAPX_BASE_GAS);
+    result.add(UNISWAPX_BASE_GAS);
+    if(adjustmentBps) {
```

Comment:
nit: should run `yarn fix` to fix linting issues

Reply:
Will do! Still a draft tho! We should block if the linting action fails tho - otherwise there is no way to enforce it